src/main/java/com/minecolonies/coremod/client/gui/WindowTownHall.java

```diff
@@ -845,7 +853,7 @@ public void onOpened()
     /**
      * Creates several statistics and sets them in the townHall GUI.
      */
-    private void createAndSetStatistics()
+    private void createAndSetStatistics_Orig()
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private "createAndSetStatistics_Orig" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUnusedPrivateMethod)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS00100)